.github/workflows/pipeline.yml

```diff
@@ -355,6 +355,46 @@ jobs:
       solana_ip: ${{ needs.prepare-infrastructure.outputs.solana_ip }}
       external_call: true
 
+  requests-report:
+    needs:
+      - prepare-infrastructure
+      - basic-tests
+      - build-image
+      - dapps-tests
+      - openzeppelin-tests
+    if: |
+      always() && 
+      contains(fromJSON('["success", "cancelled", "failure", "skipped", "completed"]'), needs.openzeppelin-tests.result) &&
+      contains(fromJSON('["success", "cancelled", "failure", "skipped", "completed"]'), needs.basic-tests.result) &&
```

Comment:
basic tests can't be skipped

I think always() is aanough here if you all statuses are good for the job

Reply:
Makes sense. I have removed check for the basic tests 👍